install/kubernetes/helm/istio/README.md

```diff
@@ -64,11 +64,9 @@ $ kubectl create ns $NAMESPACE
 
 5. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
    ```
-   $ echo -n 'admin' | base64
-   YWRtaW4=
-   $ echo -n '1f2d1e2e67df' | base64
-   MWYyZDFlMmU2N2Rm
-   $ cat <<EOF | kubectl apply -f -
+   NAMESPACE=istio-system
+   kubectl create namespace $NAMESPACE || true
```

Comment:
We do not need to create this namespace as it was already created in step 3

Reply:
I think it came in between. please see now.